### DISCUSSION TITLE
Why reactance is a required field for transmission lines?

### DISCUSSION THREAD

**Discussion by mzy2240** [ANL-CEEESA/UnitCommitment.jl#37] [Help & FAQ] [answered]
for DCOPF without phase shifting angles, I did not see a reason why we need reactance. Also I did not find a use case in the source code.

**Comment by iSoron** [comment, marked as answer]
Good catch; it should be optional. We currently only use susceptance.

**Reply by iSoron** [reply]
Removed in 58cc33ac69386983c94a04da956e673b24ada35c
